mm2src/coins/solana.rs

```diff
@@ -429,7 +429,11 @@ impl MarketCoinOps for SolanaCoin {
         unimplemented!()
     }
 
-    fn tx_enum_from_bytes(&self, _bytes: &[u8]) -> Result<TransactionEnum, String> { unimplemented!() }
+    fn tx_enum_from_bytes(&self, bytes: &[u8]) -> Result<TransactionEnum, MmError<TransactionErr>> {
+        MmError::err(TransactionErr::Plain(
+            "tx_enum_from_bytes is not supported for Solana yet.".to_string(),
```

Comment:
It could be `ParseTxError::MethodNotSupported`

Reply:
Done